machines/store.ts

```diff
@@ -67,6 +76,32 @@ export const storeMachine =
               target: 'ready',
             },
             ERROR: {
+              target: 'checkStorageInitialisation',
+            },
+          },
+        },
+        checkStorageInitialisation: {
+          invoke: {
+            src: 'checkStorageInitialisedOrNot',
+          },
+          on: {
+            ERROR: {
+              target: 'failedReadingKey',
+              actions: sendParent('ERROR'),
+            },
+            READY: {
+              target: 'generatingEncryptionKey',
+            },
+          },
+        },
+        failedReadingKey: {
+          on: {
+            TRY_AGAIN: {
+              actions: 'unsetIsReadError',
```

Comment:
unsetIsReadError -> action definition is missing

Reply:
I've added it